data/brands/amenity/fast_food.json

```diff
@@ -6906,6 +6964,21 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Pizzaman Chickenman",
+      "id": "pizzamanchickenman-0ae65d",
+      "locationSet": {"include": ["gh"]},
+      "tags": {
+        "amenity": "fast_food",
+        "brand": "Pizzaman Chickenman",
+        "brand:wikidata": "Q123097576",
+        "cuisine": "pizza; chicken",
```

Comment:
One small thing:  The `cuisine` tag can have multiple values separated by a semicolon `;` but there shouldn't be a space in there.  I removed it in aaf127e3e